src/main/java/seedu/address/logic/Logic.java

```diff
@@ -30,6 +30,9 @@
      */
     ReadOnlyAddressBook getAddressBook();
 
+    /** Returns an unmodifiable view of the filtered list of students */
```

Comment:
I think can remove 'filtered'?
"Returns an unmodifiable view of full list of students"

Reply:
Haha right! Thanks thanks! Have updated in the latest commit!